webapp/src/pages/Users/Users.tsx

```diff
@@ -1,74 +1,106 @@
+import './Users.css';
 import {
-  Header,
   IconVariant,
+  Input,
+  InputVariant,
   NavigationBar,
   RowsList,
+  RowsListProps,
+  SmallAvatar,
 } from '../../shared/components';
-import './Users.css';
+import React from 'react';
+import { USER_URL, USERS_EP_URL, USERS_URL } from '../../shared/urls';
+import { Link } from 'react-router-dom';
+
+class User {
```

Comment:
suggestion: We could convert this to an interface because we don't use the constructor anywhere. In the `transcendence-app`, we define it as a class because the swagger module requires classes to generate documentation automatically.

Reply:
done